lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -68,6 +68,10 @@ type State = {
 }
 
 const pickupDropoffOptions = [
+  {
+    value: '',
+    text: '(default)'
```

Comment:
I like the idea. Maybe say "(Inherit from route)" or something (and add i18n), and perhaps show in the `text` field what the route default is.

Reply:
I was avoiding adding "Inherit from route" or etc. because this component is shared with `Pickup_type` and `dropoff_type`, and those values do not have route level defaults. However, they do have default values if nothing is selected, that's why I chose the more generic language.

Reply:
Could we maybe add a prop that changes the string? I think `(default)` is very confusing

Reply:
Sure thing 😁

Reply:
Thoughts on the following? @binh-dam-ibigroup @miles-grant-ibigroup 
for pickup/dropoff: 
![image](https://user-images.githubusercontent.com/63798641/216131742-9fceeff9-9b74-40e7-baa9-18409d8b52d7.png)

for continuous: 
![image](https://user-images.githubusercontent.com/63798641/216131812-6eb8b2df-6128-41e4-b1a4-19eca78652cc.png)

Reply:
To me we could maybe remove the unwieldy `Inherit from route` portion by providing more help text where the user selects the route data itself

Reply:
Ok thoughts on the following? @binh-dam-ibigroup @miles-grant-ibigroup 
![image](https://user-images.githubusercontent.com/63798641/216132875-13cb28cf-6557-4f19-9151-75976b96039f.png)

![image](https://user-images.githubusercontent.com/63798641/216132906-0271876d-85ea-4170-9c09-fd76b2f52df7.png)

![image](https://user-images.githubusercontent.com/63798641/216132940-24df417d-bced-49df-8e0c-f3b838845e61.png)

Reply:
To me we could remove the unwieldy `Inherit from route` portion by adding more help text where the user enters that data in the route menu

Reply:
Okay updated per our call - this should be ready for review @binh-dam-ibigroup